lib/assert.js

```diff
@@ -151,6 +153,29 @@ x.ifError = function (err, msg) {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.snapshot = function (tree, match, snapshotStateGetter) {
```

Comment:
We should not expose these internal arguments `match` and `snapshotStateGetter` to the outside. Instead you could expose an internal base method.

```js
x._snapshot = function (tree, match, snapshotStateGetter) {

...

x.snapshot = function (tree) {
```